static/src/javascripts-legacy/projects/common/modules/experiments/ab.js

```diff
@@ -87,9 +87,12 @@ define([
     }
 
     function getActiveTests() {
-        var now = new Date();
+        var today = new Date();
+        // new Date(test.expiry) sets the expiry time to 00:00:00
+        // Using SetHours allows a test to run until the END of the expiry day
+        today.setHours(0,0,0,0);
         return TESTS.filter(function (test) {
-            var expired = (now - new Date(test.expiry)) > 0;
+            var expired = today > new Date(test.expiry);
```

Comment:
I'm super-lazy, I haven't run this. But if `today` is now at the 'beginning' of the day, doesn't this mean this test will expire at the _start_ of `today` - aren't we aiming for it to expire at the end?